### ISSUE TITLE
said it was successful but nothing showing as watched

### ISSUE THREAD

**Issue by KwisatzJim** [wilmardo/migrate-plex-to-jellyfin#16]
Hi
I just discovered Jellyfin today and installed it on my iMac.  I just ran the migrate.py script and it said Succesfully migrated 5609 items, but when I look in my Jellyfin library nothing is showing as watched.
Is there something else I need to do?
thanks!

**Comment by awb1015** [comment]
I had the exact same issue

**Comment by wilmardo** [comment]
Yeah this repo needs some work haha. You need to edit the libraries in the script itself since it biased towards my library names.

https://github.com/wilmardo/migrate-plex-to-jellyfin/issues/9

It was just a dump of my personal script but I hope to find some time to make it more generic since it seems a popular migration :)

**Comment by wilmardo** [comment]
I am closing this since users have run the scripts succesfully the last couple months.
